### DISCUSSION TITLE
blog/git-butler-example

### DISCUSSION THREAD

**Discussion by giscus** [korECM/tw-blog#18] [Comments]
# blog/git-butler-example

Scott Chacon GitButler를 어떻게 사용할 수 있을지 살펴보았습니다

https://www.korecmblog.com/blog/git-butler-example

**Comment by ProtossDragoon** [comment]
글이 정말 탄탄하게 잘 써진것같아서 댓글 남겨요 종원님 🫡👍 나도 써봐야겠다

**Comment by ProtossDragoon** [comment]
이제 써보고싶어서 왔다가 잘 배워갑니다.
